support/cas-server-support-consent-webflow/src/main/java/org/apereo/cas/web/consent/config/CasConsentReviewConfiguration.java

```diff
@@ -0,0 +1,151 @@
+    private ConsentRepository consentRepository;
+
+    @Autowired
+    @Qualifier("consentEngine")
+    private ConsentEngine consentEngine;
+
+    @Bean
+    @RefreshScope
+    public CasConsentReviewController casConsentReviewController() {
+        return new CasConsentReviewController(consentRepository, consentEngine, casConsentPac4jConfig(),
+                casProperties.getServer().getPrefix().concat("/consent"));
+    }
+
+    @Bean
+    @RefreshScope
+    public Config casConsentPac4jConfig() {
+        final CasConfiguration conf = new CasConfiguration(casProperties.getServer().getLoginUrl());
+        
+        final CasClient client = new CasClient(conf);
+        client.setName(CAS_CONSENT_CLIENT);
+        client.setCallbackUrl(casProperties.getServer().getPrefix().concat("/consent/callback"));
+        client.setAuthorizationGenerator(new DefaultCasAuthorizationGenerator<>());
+        client.setIncludeClientNameInCallbackUrl(false);
+        
+        final Clients clients = new Clients(client);
+        clients.setDefaultClient(client);
+        
+        final Config config = new Config(clients);
+        config.setAuthorizer(new IsAuthenticatedAuthorizer());
+        config.setCallbackLogic(new DefaultCallbackLogic());
+        config.setLogoutLogic(new DefaultLogoutLogic());
+        
+        // get role authorizer from admin pages for smooth integration
+        final Map<String, Authorizer> adminAuthorizers = casAdminPagesPac4jConfig.getAuthorizers();
+        final String auth = RequireAnyRoleAuthorizer.class.getSimpleName();
+        if (adminAuthorizers.containsKey(auth)) {
+            config.addAuthorizer(auth, adminAuthorizers.get(auth));
+        }
+        return config;
+    }
+
+    @Bean
+    @RefreshScope
+    public CasSecurityInterceptor casConsentReviewSecurityInterceptor() {
+        return new CasSecurityInterceptor(casConsentPac4jConfig(), CAS_CONSENT_CLIENT,
+                "securityHeaders,csrfToken,".concat(IsAuthenticatedAuthorizer.class.getSimpleName()));
+    }
+    
+    /**
+    * Initialize consent service.
+    */
+    @PostConstruct
+    protected void registerConsentService() {
+        final Service callbackService = this.webApplicationServiceFactory.createService(
+                casProperties.getServer().getPrefix().concat("/consent.*"));
+        if (!this.servicesManager.matchesExistingService(callbackService)) {
+            LOGGER.debug("Initializing consent service [{}]", callbackService);
+
+            final RegexRegisteredService service = new RegexRegisteredService();
+            service.setEvaluationOrder(0);
+            service.setName("CAS Consent Review");
+            service.setDescription("Review consent decisions for attribute release");
+            service.setServiceId(callbackService.getId());
+            
+            // disable consent for this service
+            final ReturnAllAttributeReleasePolicy policy = new ReturnAllAttributeReleasePolicy();
```

Comment:
Why this?

Reply:
Are you refering to the specific AttributeReleasePolicy, or to disabling consent for this service?

Reply:
Sorry; should have been clearer; the attribute release policy specifically.

Reply:
Haven't thought about it. I somehow thought this was the default. No extra attributes are needed, so I'll review this.

Reply:
Gotcha. I would either set this to deny attributes explicitly, or leave it as default (which probably is the better option).